.github/workflows/tests.yml

```diff
@@ -95,6 +95,10 @@ jobs:
       - name: Setup Poetry
         uses: matrix-org/setup-python-poetry@v1
         with:
+          # We set this as our minimum supported Python version. We use the oldest
+          # Python version because later Python versions can include some overloads
+          # which don't work in the older versions.
+          python-version: '3.8'
```

Comment:
Should we also include `python_version = 3.8` in `mypy.ini`? https://mypy.readthedocs.io/en/stable/config_file.html#platform-configuration

Is that a sufficient alternative to changing the Python version here?

Reply:
Defining in `mypy.ini` seems to work great since now you can see the lints locally regardless of what Python version you're running

Updated :fast_forward: